src/ServiceControl/Recoverability/Grouping/FailureGroupsApi.cs

```diff
@@ -84,7 +84,8 @@ dynamic GetAllGroups(string classifier)
                             First = failureGroup.First,
                             Last = failureGroup.Last,
                             RetryStatus = summary != null ? "In Progress" : null,
-                            NumberOfRetryMessagesRemaining = summary?.MessagesRemaining
+                            NumberOfRetryMessagesRemaining = summary?.MessagesRemaining,
+                            IsPossiblyAnIncompleteRetry = summary.IsPossiblyIncomplete
```

Comment:
No, all the documents could have been marked and SC could have crashed before updating the batch status